web/lib/store/auth/index.js

```diff
@@ -50,15 +50,15 @@ export const loginAction = createAsyncThunk('auth/loginAction', async ({ params,
     dispatch(setAuthParams(params))
     localStorage.setItem('authParams', JSON.stringify(params))
 
-    const url = isProdEnv ? getState().auth.oauthUrl : devOauthUrl
+    const url = getState().auth.oauthUrl
```

Comment:
@ch3yne  No need to make conditional judgments here?

Reply:
Because the backend has handled the cross-origin issue, there's no need to do proxy forwarding now. Just request the original address directly.